gemini/function-calling/function_calling_data_structures.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>Has the <code>bigframes&lt;1.0.0</code>  bug been fixed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes, this was fixed in google-cloud-aiplatform==1.45.0, and errors that were happening were solved when this was pushed into the latest container images in Colab. 👍